app/src/main/java/com/snatik/matches/common/Memory.java

```diff
@@ -3,26 +3,81 @@
 import android.content.Context;
 import android.content.SharedPreferences;
 import android.content.SharedPreferences.Editor;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import java.util.Map;
+
+import static android.R.attr.id;
+import static android.R.id.edit;
+import static android.content.Context.MODE_PRIVATE;
 
 
 public class Memory {
 
 	private static final String SHARED_PREFERENCES_NAME = "com.snatik.matches";
 	private static String highStartKey = "theme_%d_difficulty_%d";
+	private static String bestTimeKey = "themeTime_%d_difficultyTime_%d";
```

Comment:
please change the key to lower case: `theme_%d_time_%d`

Reply:
Could you explain why this should be changed please?

Reply:
Just to follow the existing code styling. This is important, when you update existing code. There is no other explanation beside this. It can work with and without capital cases.

Reply:
Okay. Did it